libs/server-sdk/src/config/config_builder.cpp

```diff
@@ -40,7 +34,7 @@ tl::expected<Config, Error> ConfigBuilder::Build() const {
     if (sdk_key.empty()) {
         return tl::make_unexpected(Error::kConfig_SDKKey_Empty);
     }
-    auto offline = offline_.value_or(Defaults::Offline());
```

Comment:
Ignore this. Had some back-and-forth thoughts about offline mode; will add it back in as a top-level config since it's convenient / status quo.